src/minimal/partials/header.hbs

```diff
@@ -3,7 +3,7 @@
         <div class="container">
             <div class="table-wrap">
                 <div class="table-cell">
-                    <strong>{{project.name}}</strong>
+                    <strong><a href="{{relativeURL "index.html"}}">{{project.name}}</a></strong>
```

Comment:
Actually, I see the default theme is set up the same way, I'll keep it consistent.